packages/js-commons/src/core/local-files.ts

```diff
@@ -188,15 +286,31 @@ export const saveArtifact = async (
     pathToWrite = `${artifact.name}.yaml`;
   }
 
-  await writeFile(join(artifactSubDir, pathToWrite), artifact.content, "utf-8");
+  if (
+    artifact.type !== EnsembleDocumentType.Asset &&
+    artifact.type !== EnsembleDocumentType.Font
+  ) {
```

Comment:
Don't need this anymore?

Reply:
we still do, writing asset & font is dealt in `localStoreAsset` method, `saveArtifact` is called only for non-asset artifacts